src/btree/bt_vrfy.c

```diff
@@ -901,6 +903,83 @@ __verify_page_content_int(
     return (0);
 }
 
+/*
+ * __verify_page_content_fix --
+ *     Verify the page's content. Like __verify_page_content_leaf but for FLCS pages.
+ */
+static int
+__verify_page_content_fix(
+  WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK_ADDR *parent, WT_VSTUFF *vs)
+{
+    WT_CELL *cell;
+    WT_CELL_UNPACK_KV unpack;
+    WT_DECL_RET;
+    WT_PAGE *page;
+    uint64_t start_ts;
+    uint32_t cell_num, numtws, recno_offset, tw;
+    uint8_t *p;
+
+    page = ref->page;
+
+    /*
+     * If a tree is empty (just created), it won't have a disk image; if there is no disk image,
+     * we're done.
+     */
+    if (page->dsk == NULL) {
+        WT_ASSERT(session, page->entries == 0);
+        return (0);
+    }
+
+    /* Count the keys. */
+    vs->records_so_far += page->entries;
+
+    /* Walk the time windows, if there are any. */
+    numtws = WT_COL_FIX_TWS_SET(page) ? page->pg_fix_numtws : 0;
+    for (tw = 0; tw < numtws; tw++) {
+        /* The printable cell number for the value is 2x the entry number (tw) plus 1. */
+        cell_num = tw * 2 + 1;
+
+        cell = WT_COL_FIX_TW_CELL(page, &page->pg_fix_tws[tw]);
+        __wt_cell_unpack_kv(session, page->dsk, cell, &unpack);
+        /* If this fails it means the index was built wrong. */
+        WT_ASSERT(session, unpack.type == WT_CELL_VALUE);
```

Comment:
That will allow it to point at the key cells, which probably isn't good. I'll make it fail gracefully though since if the page contains e.g. an overflow value instead of a normal value, which it's not supposed to, it'll only get caught here. Also I just noticed that this code should and doesn't check that the value itself is empty. Fixed both of these points.